### DISCUSSION TITLE
Please help me, I'm new to this. Provide me with the code for Schlumberger VES inversion with field data

### DISCUSSION THREAD

**Discussion by ferryatm** [gimli-org/pyGIMLi#672] [Q&A]
Please help me, I'm new to this. Provide me with the code for Schlumberger VES inversion with field data.

| MN/2 (m) | AB/2 (m) | Rho Apparent (Ohm.m) |
|----------|----------|-------------------------|
|   0.5    |   1.5    |         5.635211        |
|   0.5    |   2.0    |         4.612248        |
|   0.5    |   2.5    |         4.513198        |
|   0.5    |   3.0    |         4.17725         |
|   0.5    |   4.0    |         5.30145         |
|   0.5    |   5.0    |         4.7350773       |
|   0.5    |   6.0    |         4.3606916       |
|   1.0    |   6.0    |         4.3606916       |
|   1.0    |   7.0    |         4.614735575     |
|   1.0    |   8.0    |         5.28781021      |
|   1.0    |   9.0    |         5.463961275     |
|   1.0    |   10.0   |         5.334367416     |
|   1.0    |   12.0   |         5.871748295     |
|   1.0    |   15.0   |         6.864989891     |
|   5.0    |   15.0   |         6.864989891     |
|   5.0    |   20.0   |         8.31374329      |
|   5.0    |   25.0   |         10.45239054     |
|   5.0    |   30.0   |         11.65316963     |
|   10.0   |   30.0   |         11.65316963     |
|   10.0   |   40.0   |         13.94573528     |
|   10.0   |   50.0   |         16.34242351     |
|   10.0   |   60.0   |         17.40842728     |
|   20.0   |   60.0   |         17.40842728     |
|   20.0   |   70.0   |         20.31610704     |
|   20.0   |   80.0   |         19.13295652     |
|   20.0   |   90.0   |         20.17732947     |
|   20.0   |   100.0  |         20.20464984     |

**Comment by ferryatm** [comment]
please help me

**Comment by ferryatm** [comment]
[Datasheet TA.xlsx](https://github.com/gimli-org/gimli/files/14606941/Datasheet.TA.xlsx)

**Comment by NoteboomM** [comment]
Still looking for help on this? The tutorial at [https://www.pygimli.org/_tutorials_auto/3_inversion/plot_2-dc1dblock.html#sphx-glr-tutorials-auto-3-inversion-plot-2-dc1dblock-py](url) should get you pretty close. All you need to do is replace the forward modelling section with a few lines of code to define arrays of ab/2, mn/2 and rho.

**Comment by halbmy** [comment]
Closing as duplicate of the solved issue #673
